spec/game_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rspec'
+require_relative '../classes/game'
+require_relative '../classes/item'
+
+describe Game do
+  let(:game_params) do
+    {
+      id: 1,
+      publish_date: '2020-01-01',
+      last_played_at: '2023-01-01',
+      multiplayer: true
+    }
+  end
+
+  subject(:game) { described_class.new(game_params) }
+
+  before do
+    allow(Date).to receive(:parse).and_call_original
+    allow(Date).to receive(:parse).with('2023-01-01').and_return(Date.new(2023, 1, 1))
+  end
+
+  describe '#initialize' do
+    it 'sets the multiplayer attribute' do
+      expect(game.multiplayer).to eq(game_params[:multiplayer])
+    end
+
+    it 'sets the last_played_at attribute' do
+      expect(game.last_played_at).to eq(Date.new(2023, 1, 1))
+    end
+  end
```

Comment:
- A 'Game' instance, only has two properties? ('multiplayer' and 'last_played_at')
- Isn't 'Game' a child class?